app/src/components/MiniGlobe.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MiniGlobeStyles from 'styles/components/miniGlobe.scss';
+import { geoOrthographic, geoPath } from 'd3-geo'; // eslint-disable-line
+import { feature } from 'topojson-client';
+import { MINI_GLOBE_SETTINGS } from 'config';
+
+const jsonData = require('assets/topoJson/ne_110m_land.json');
+
+class MiniGlobe extends Component {
+
+  constructor(props) {
+    super();
+    this.worldData = feature(jsonData, jsonData.objects.land).features;
+    const viewportWidth = props.viewportWidth;
+    const viewportHeight = props.viewportHeight;
+    const size = this.markerSize(props.zoom, viewportWidth, viewportHeight);
+
+    this.setState({
```

Comment:
?

Reply:
This is the size of the red marker to show the covered map area